src/values/NativeFunctionValue.js

```diff
@@ -83,6 +83,10 @@ export default class NativeFunctionValue extends FunctionValue {
     }
   }
 
+  hasDefaultLength(): boolean {
+    return true;
+  }
+
```

Comment:
I think it's not needed right now, as a native functions are intrinsics, and we add entries to the generator to mutate their properties.

To make sure we catch the case when this is no longer true, you could replace the body of this function with `throw new Error("TODO");` as there's really no way to test this right now. Or you do the right implementation, which is probably something like `return this.length === this.getLength()`.